example/cumulus-tf/variables.tf

```diff
@@ -19,6 +19,10 @@ variable "cmr_provider" {
 variable "cmr_username" {
   type = string
 }
+variable "bucket_map_key" {
```

Comment:
nit: precede with a blank line for consistency

Reply:
1624d7e07 -- fixed spacing too, whileI was at it.